pkg/jmxfetch/jmxfetch.go

```diff
@@ -75,6 +75,15 @@ type JMXFetch struct {
 	stopped            chan struct{}
 }
 
+// JMXFetch supports different way of reporting the data it has fetched.
+type JMXReporter string
+
+var (
+	ReporterDefault JMXReporter = "statsd"
```

Comment:
Nit: We most likely need to add the `jmx` build tag to vet. You are supposed to have to add comments for those exported vars I think 🤔